### PR TITLE
fix some errors in generating

### PR DESCRIPTION
**Pull request by chillancezen**

### CODE REVIEW

**Comment by chillancezen** [comment]
```
generated config fact in output/v6.0.0/fortios_configuration_fact.py
Traceback (most recent call last):
  File "./scripts/generate_modules.py", line 313, in <module>
    jinjaExecutor(arg)
  File "./scripts/generate_modules.py", line 289, in jinjaExecutor
    generate_cofiguration_fact_rst(fgt_sch_results, fgt_schema['version'])
  File "/home/linky/ansible/fortios-ansible-generator/scripts/generate_modules_utility.py", line 25, in generate_cofiguration_fact_rst
    data = template.render(selectors=selectors)
  File "/home/linky/.local/lib/python2.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/linky/.local/lib/python2.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "ansible_templates/configuration_fact.rst.j2", line 49, in top-level template code
    <li><span class="li-normal">{{selector}}</span> {% if selectors[selector]['mkey'] %} <span class="li-required">param: {{ selectors[selector]['mkey'] }}</span>  <span class="li-required">type: {{ selectors[selector]['mkey_type'] }}</span>{% endif%}</li>
  File "/home/linky/.local/lib/python2.7/site-packages/jinja2/environment.py", line 452, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: dict object has no element (u'alertemail_setting', {'mkey_type': None, 'mkey': None})
generating online document v6.0.0
```

File: ansible_templates/configuration_fact.rst.j2 [hunk 1]
```diff
@@ -44,7 +44,7 @@ Parameters
         </h2>
         <div class="content">
         <ul class="ul-self">
-        {% for selector in selectors | dictsort(false) -%}
```
Comment (JonTheNiceGuy) [inline ansible_templates/configuration_fact.rst.j2:47]: Apologies. This should have been sorting the selectors. I'm guessing the fact you've removed this mean that [my source for this](https://stackoverflow.com/a/61415554/5738) didn't work?

Comment (chillancezen) [reply ansible_templates/configuration_fact.rst.j2:47]: No worry Jon, it just prompts traceback as stated in https://github.com/fortinet-ansible-dev/fortios-ansible-generator/pull/20#issuecomment-745134135 

for the urgency to update sphinx online document, I removed this sorter. 

I do know sorting selector will be of great benefit. 

Jon, could you have a chance to help investigate this issue?
(steps to generate: https://github.com/fortinet-ansible-dev/fortios-ansible-generator/pull/19#issuecomment-745087649)

thanks,
LInk

Comment (JonTheNiceGuy) [reply ansible_templates/configuration_fact.rst.j2:47]: Ugh, I'm having big problems with building the docs. I'm getting this:

```
(FortiOS_Generator) user@host:~/FortiOS-Ansible-Generator (patch-2 *%)$ ~/FortiOS_Generator/bin/python ./scripts/generate_modules.py 
args ['./scripts/generate_modules.py']

Parsing schema:
Module name: system_vdom
Iteration:0, Schema position: 0
Traceback (most recent call last):
  File "./scripts/generate_modules.py", line 313, in <module>
    jinjaExecutor(arg)
  File "./scripts/generate_modules.py", line 250, in jinjaExecutor
    renderModule(fgt_sch_results[i],
  File "./scripts/generate_modules.py", line 114, in renderModule
    schema['schema'] = hyphenToUnderscore(schema['schema'])
  File "./scripts/generate_modules.py", line 58, in hyphenToUnderscore
    data[k] = hyphenToUnderscore(v)
  File "./scripts/generate_modules.py", line 58, in hyphenToUnderscore
    data[k] = hyphenToUnderscore(v)
  File "./scripts/generate_modules.py", line 58, in hyphenToUnderscore
    data[k] = hyphenToUnderscore(v)
  File "./scripts/generate_modules.py", line 62, in hyphenToUnderscore
    elif isinstance(data, unicode):
NameError: name 'unicode' is not defined
```

This is WSL1 Ubuntu 20.04 on Windows 10. The FortiOS_Generator virtualenv has ['ansible', 'jinja2', 'unicode'] installed.

Any thoughts?

Comment (chillancezen) [reply ansible_templates/configuration_fact.rst.j2:47]: this is also a legacy issue,

we need `python2` to generate the modules.
will you be able to install python2 by `apt install python2`
so sorry for the delay.

**Pull request merged**